pkgs/love2d/example/assets/modern_pong/states/mainMenu.lua

```
@@ -0,0 +1,145 @@
+function MainMenu:update(dt)
+    if MainMenu.delay and MainMenu.delay > 0 then
+        MainMenu.delay = MainMenu.delay - dt
+        return
+    end
+
+    -- Reset delay to nil to indicate the delay period is over
+    MainMenu.delay = nil
+
+    local x, y = love.mouse.getPosition()
+
+    for i, button in ipairs(buttons) do
```

Luacheck (1.2.0)

[warning] unused loop variable 'i'

(W213)